packages/core/src/preview.ts

```diff
@@ -89,16 +98,23 @@ export const getPreviewString = (
       skippingArray.push(skippingReason)
       for (let i = 0; i < skipping.length; i++) {
         const element = skipping[i]
-        const functionCallStr = isDecodedAction(element)
-          ? prettyFunctionCall(
-              element.referenceName,
-              element.address,
-              element.functionName,
-              element.variables,
-              5,
-              3
-            )
-          : prettyRawFunctionCall(element.to, element.data)
```

Comment:
Needed to refactor this to account for the new `PreviewElement` type, `SystemDeploymentElement`